serpentTools/parsers/bumat.py

```diff
@@ -1,5 +1,5 @@
 """Parser responsible for reading the ``*bumat<n>.m`` files"""
-
+debugserpentoutput=False
```

Comment:
We have our own debug/logging functionality controlled by the [`verbosity`](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/develop/serpentTools/settings/__init__.py#L34) setting that controls the [logging module](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/develop/serpentTools/settings/messages.py).
Some of my comments below will, _hopefully_, explain the `messages` module